plugins/modules/proxysql_cluster.py

```diff
@@ -0,0 +1,443 @@
+        query_string += ")"
+
+        cursor.execute(query_string, query_data)
+        return True
+
+    def update_server_config(self, cursor):
+        query_string = "UPDATE proxysql_servers"
+
+        cols = 0
+        query_data = []
+
+        for col, val in iteritems(self.config_data):
+            if val is not None:
+                cols += 1
+                query_data.append(val)
+                if cols == 1:
+                    query_string += " SET " + col + "= %s,"
+                else:
+                    query_string += " " + col + " = %s,"
+
+        query_string = query_string[:-1]
+        query_string += ("WHERE hostname = %s AND port = %s")
+
+        query_data.append(self.hostname)
+        query_data.append(self.port)
+
+        cursor.execute(query_string, query_data)
+        return True
+
+    def delete_server_config(self, cursor):
+        query_string = (
+            "DELETE FROM proxysql_servers "
+            "WHERE hostname = %s "
+            "AND port = %s"
+        )
+
+        query_data = [
+            self.hostname,
+            self.port
+        ]
+
+        cursor.execute(query_string, query_data)
+        return True
+
+    def manage_config(self, cursor, state):
+        if state:
+            if self.save_to_disk:
+                save_config_to_disk(cursor)
+            if self.load_to_runtime:
+                load_config_to_runtime(cursor)
+
+    def create_server(self, check_mode, result, cursor):
+        if not check_mode:
+            result['changed'] = self.create_server_config(cursor)
+            result['msg'] = "Added server to proxysql cluster"
+            result['server'] = self.get_server_config(cursor)
+            self.manage_config(cursor, result['changed'])
+        else:
+            result['changed'] = True
+            result['msg'] = (
+                "Server would have been added to "
+                "proxysql cluster, however check_mode "
+                "is enabled."
+            )
+
+    def update_server(self, check_mode, result, cursor):
+        if not check_mode:
+            result['changed'] = self.update_server_config(cursor)
+            result['msg'] = "Updated server in proxysql cluster"
+            result['server'] = self.get_server_config(cursor)
+            self.manage_config(cursor, result['changed'])
+        else:
+            result['changed'] = True
+            result['msg'] = (
+                "Server would have been updated in "
+                "proxysql cluster, however check_mode "
+                "is enabled."
+            )
+
+    def delete_server(self, check_mode, result, cursor):
+        if not check_mode:
+            result['server'] = self.get_server_config(cursor)
+            result['changed'] = self.delete_server_config(cursor)
+            result['msg'] = "Deleted server from proxysql cluster"
+            self.manage_config(cursor, result['changed'])
+        else:
+            result['changed'] = True
```

Comment:
```suggestion
            result['changed'] = True
```
Same things about check_mode from the other PR

Reply:
Please, see https://github.com/ansible-collections/community.proxysql/pull/26#issuecomment-838312886 and https://github.com/ansible-collections/community.proxysql/pull/26#issuecomment-838340843